documentation/modules/rn-2.3.adoc

```diff
@@ -71,3 +71,9 @@ The error status message for a VM with no operating system on the *Migration pla
 .Log archive file includes logs of a deleted migration plan or VM
 
 If you delete a migration plan and then run a new migration plan with the same name or if you delete a migrated VM and then remigrate the source VM, the log archive file created by the {project-short} web console might include the logs of the deleted migration plan or VM. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764[*BZ#2023764*])
+
+.{rhv-short} Migration of virtual machines with encrypted partitions fails during conversion
+The problem occures for both vSphere amd {project-short} migrations. 
+
+.{project-short} 2.3.4 only: Duplication of migration plan fails in storage mapping stage if source provider is {rhv-short}.
```

Comment:
we've seen it also in the network mapping stage, it would be better to mention both cases